app/javascript/test/Jobs.test.tsx

```diff
@@ -22,117 +41,284 @@ Object.defineProperty(window, 'matchMedia', {
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'PA',
+      applicationCloseDate: 'September 20, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 2',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 2',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'PA',
+      applicationCloseDate: 'September 21, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 3',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 3',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'PA',
+      applicationCloseDate: 'September 22, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 4',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 4',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'PA',
+      applicationCloseDate: 'September 23, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 5',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 5',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'Per Year',
+      applicationCloseDate: 'September 24, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 6',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 6',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'Per Hour',
+      applicationCloseDate: 'September 24, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 7',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 7',
+      minimumPay: 122000,
+      maximumPay: 150000,
+      rateIntervalCode: 'Without Compensation',
+      applicationCloseDate: 'September 24, 2023'
+    },
+    {
+      positionTitle: 'Contract Specialist 8',
+      positionUri: 'https://www.usajobs.gov/GetJob/ViewDetails/690037300',
+      positionLocation: 'Multiple Locations',
+      organizationName: 'Office of Acquisition and Logistics 8',
+      minimumPay: 0,
+      maximumPay: 150000,
+      rateIntervalCode: 'PA',
+      applicationCloseDate: 'September 24, 2023'
+    }
+  ],
+  agencyName: 'USA.gov',
+  query: 'jobs',
+  affiliate: 'boos_affiliate',
+  vertical: 'web'
+};
+
+const jobsProps2 = { 
```

Comment:
what's the difference with `jobsProps`? can we get away using one `jobsProps`?

Reply:
renamed the variables.